1.1.0 FTBFS

/kind bug

**Steps to reproduce the issue:**

1. Build with tags: `seccomp apparmor containers_image_ostree_stub btrfs_noversion exclude_graphdriver_btrfs`

**Describe the results you received:**

```
src/github.com/containers/libpod/cmd/podman/rmi.go:39:8: undefined: iopodman.ImageNotFound
```

**Describe the results you expected:**

--- a/cmd/podman/rmi.go
+++ b/cmd/podman/rmi.go
@@ -5,8 +5,6 @@ import (
 	"os"
 
 	"github.com/containers/libpod/cmd/podman/cliconfig"
-	"github.com/containers/libpod/cmd/podman/varlink"
-	"github.com/containers/libpod/libpod/image"
 	"github.com/containers/libpod/pkg/adapter"
 	"github.com/containers/storage"
 	"github.com/pkg/errors"
@@ -30,17 +28,6 @@ var (
   podman rmi c4dfb1609ee2 93fd78260bd1 c0ed59d05ff7`,
 	}
 )
-
-func imageNotFound(err error) bool {
-	if errors.Cause(err) == image.ErrNoSuchImage {
-		return true
-	}
-	switch err.(type) {
-	case *iopodman.ImageNotFound:
-		return true
-	}
-	return false
-}
 
 func init() {
 	rmiCommand.Command = _rmiCommand
@@ -80,7 +67,7 @@ func rmiCmd(c *cliconfig.RmiValues) error {
 			if errors.Cause(err) == storage.ErrImageUsedByContainer {
 				fmt.Printf("A container associated with containers/storage, i.e. via Buildah, CRI-O, etc., may be associated with this image: %-12.12s\n", img.ID())
 			}
-			if !imageNotFound(err) {
+			if !adapter.IsImageNotFound(err) {
 				failureCnt++
 			}
 			if lastError != nil {
@@ -135,7 +122,7 @@ func rmiCmd(c *cliconfig.RmiValues) error {
 			newImage, err := runtime.NewImageFromLocal(i)
 			if err != nil {
 				if lastError != nil {
-					if !imageNotFound(lastError) {
+					if !adapter.IsImageNotFound(lastError) {
 						failureCnt++
 					}
 					fmt.Fprintln(os.Stderr, lastError)
@@ -147,7 +134,7 @@ func rmiCmd(c *cliconfig.RmiValues) error {
 		}
 	}
 
-	if imageNotFound(lastError) && failureCnt == 0 {
+	if adapter.IsImageNotFound(lastError) && failureCnt == 0 {
 		exitCode = 1
 	}
 

--- a/pkg/adapter/runtime.go
+++ b/pkg/adapter/runtime.go
@@ -333,3 +333,11 @@ func (r *LocalRuntime) LoadImage(ctx context.Context, name string, cli *cliconfi
 	}
 	return r.Runtime.LoadImage(ctx, name, cli.Input, writer, cli.SignaturePolicy)
 }
+
+// IsImageNotFound checks if the error indicates that no image was found.
+func IsImageNotFound(err error) bool {
+	if errors.Cause(err) == image.ErrNoSuchImage {
+		return true
+	}
+	return false
+}

--- a/pkg/adapter/runtime_remote.go
+++ b/pkg/adapter/runtime_remote.go
@@ -796,3 +796,15 @@ func (r *LocalRuntime) LoadImage(ctx context.Context, name string, cli *cliconfi
 	}
 	return names, nil
 }
+
+// IsImageNotFound checks if the error indicates that no image was found.
+func IsImageNotFound(err error) bool {
+	if errors.Cause(err) == image.ErrNoSuchImage {
+		return true
+	}
+	switch err.(type) {
+	case *iopodman.ImageNotFound:
+		return true
+	}
+	return false
+}

